src/irr/asset/CMeshManipulator.cpp

```diff
@@ -868,7 +869,7 @@ void CMeshManipulator::copyMeshBufferMemberVars<ICPUMeshBuffer>(ICPUMeshBuffer*
     auto ixBinding = _src->getIndexBufferBinding()[0];
 	_dst->setIndexBufferBinding(
 		std::move(ixBinding)
-	);/*
+	);/* @Crisspl wtf with the commenting out???
```

Comment:
see `createMeshBufferDuplicate`, also note taht `copyMeshBufferMemberVars` is private